src/rsz/src/RepairSetup.cc

```diff
@@ -569,6 +579,30 @@ bool RepairSetup::swapPins(PathRef* drvr_path,
   return false;
 }
 
+// Remove driver if
+// 1) it is a buffer without attributes like dont-touch
+// 2) it improves setup slack
+// 3) it doesn't create new max fanout violations
+// 4) it doesn't create new max cap violations
+// 5) it doesn't create new max slew violations
+bool RepairSetup::removeDrvr(PathRef* drvr_path,
+                             LibertyCell* drvr_cell,
+                             int drvr_index,
+                             PathExpanded* expanded)
+{
+  /* TODO: remove driver only if slack improves and no new violations
+  if (drvr_cell && drvr_cell->isBuffer()) {
+    Pin* drvr_pin = drvr_path->pin(this);
+    Instance* drvr = network_->instance(drvr_pin);
+    if (remove_buffer_->removeBuffer(drvr)) {
+      return true;
+    }
+  }
+  */
```

Comment:
Do you plan to remove this before merging?

Reply:
Removed